moveos/moveos-client/src/client.rs

```diff
@@ -0,0 +1,178 @@
+struct SendTxRequest {
+    sender_address: String,
+    public_key: Vec<u8>,
+    program: Vec<u8>,
+    sequencer_number: u64,
+    expiration_time: u64,
+    signature: Vec<u8>,
+}
+
+struct SendTxnResponse {
+    // The order of a user's transactions in a block.
+    order: u128,
+    // Transaction Hash
+    txn: String,
+    // The signature of the sequencer on this order information.
+    txn_order_signature: Vec<u8>,
+}
+
+struct VerifyTxnRequest {
```

Comment:
sequencer has no ability to verify "anything". The power of verfication is always be Layer1.

Sequencer could give the TxnOrderWitness, so that should be GetTxnOrderWitnessRequest. And the request body should be TxnOrder tuple I mentioned in the last comment. (with signature, only signature could trigger witness generating)